app/config/wallabag.yml

```diff
@@ -26,6 +26,7 @@ wallabag_core:
     fetching_error_message: |
         wallabag can't retrieve contents for this article. Please <a href="http://doc.wallabag.org/en/user/errors_during_fetching.html#how-can-i-help-to-fix-that">troubleshoot this issue</a>.
     api_limit_mass_actions: 10
+    domain_name: '%domain_name%'
```

Comment:
Also, I don't think you need to include the domain here. We can use it from the parameter directly.